common/services/github.js

```diff
@@ -342,6 +343,18 @@ async function _getPullRequestsDetailsByCommitShas({ repoOwner, repoName, commit
   return pullRequestsForCommitShaFilteredDetails;
 }
 
+async function getFilesModifiedBeetwenTwoReleases({ endpoint }) {
+  const octokit = _createOctokit();
+  const { data } = await octokit.repos.compareCommits(endpoint);
+  return { files: data.files, commits: data.commits};
+}
+
+function hasConfigBeenModified(files) {
+    const result = files.filter(f => f.filename === 'api/lib/config.js');
```

Comment:
```suggestion
    const result = files.filter(f => f.filename === 'api/src/shared/config.js');
```

Le fichier de config a été déplacé entre temps